### DISCUSSION TITLE
Bulk Editing

### DISCUSSION THREAD

**Discussion by RhysCrawford** [statamic/cms#7939] [Q&A]
Hello, 

I am a new Statamic user, coming into an existing infrastructure.  Every year pricing changes have to be made, which affects almost 300 static pages.   Currently, the process is to manually update the "price" field on each of those pages.  It seems to me that there should be some kind of bulk edit option where I can say 'Update [price] to $xxx where [program] is yyy and [location] is zzz"  (program and location are two other fields in the same blueprint)

Is there any kind of add-on or functionality to do this with Statamic?  We have discussed moving from a flat-file implementation to a SQL implementation, in which case I can edit the database directly to make this change.  But before we make that change, is there a way to do this within Statamic?

Thank you!

**Comment by duncanmcclean** [comment]
You could likely write a query to find the ones needing updated, then do the update for each of them: https://statamic.dev/content-queries#overview
